tests/rados/alerts.py

```diff
@@ -0,0 +1,79 @@
+"""Module to test ceph health alerts"""
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Prepares the cluster & runs alerts test Scenarios.
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster
+        kw: Args that need to be passed to the test for initialization
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    # List of alerts to be tested
+    alert_list = ["HEALTH_ERR", "HEALTH_WARN"]
+    for alert in alert_list:
+        if alert == "HEALTH_ERR":
+            if not generate_health_error_alert(alert, cephadm):
+                return 1
+    return 0
```

Comment:
Please revert the config changes made during the test.

Reply:
@pdhiran Yes done that in that call itself refer line no 70 I am resetting default config.